src/internet/podcasts/podcastparser.cpp

```diff
@@ -291,7 +291,13 @@ void PodcastParser::ParseOutline(QXmlStreamReader* reader,
           // Parse the feed and add it to this container
           Podcast podcast;
           podcast.set_description(attributes.value("description").toString());
-          podcast.set_title(attributes.value("text").toString());
+          QString title = attributes.value("text").toString();
```

Comment:
do we need to declare a variable to store this ? what about 
if (attributes.value("text").toString().isEmpty())
   podcast.set_title(attributes.value("title").toString());
else
   podcast.set_title(attributes.value("text").toString());